start_streaming.bat

```diff
@@ -21,8 +21,7 @@ IF "%USE_RTSS%"=="true" (
 )
 
 :: Enable the virtual display
-PNPUTIL /enable-device /deviceid "root\iddsampledriver"
-PNPUTIL /enable-device /deviceid "MONITOR\LNX0000"
+powershell -NoProfile -ExecutionPolicy Bypass -Command "$device = Get-PnpDevice | Where-Object {$_.FriendlyName -eq 'Virtual Display Driver by MTT'}; if ($device) { Enable-PnpDevice -InstanceId $device.InstanceId -Confirm:$false }"
```

Comment:
Unfortunately this doesn't work with the Windows 11 version of VDD supporting HDR, as the device information is different, including the friendly description you're using for the lookup.

```
Instance ID:                ROOT\DISPLAY\0000
Device Description:         IddSampleDriver Device HDR
Class Name:                 Display
Class GUID:                 {4d36e968-e325-11ce-bfc1-08002be10318}
Manufacturer Name:          <Your manufacturer name>
Status:                     Started
Driver Name:                oem93.inf
```

If you want to keep the lookup approach, you'd need to search by both names so it works for both versions of VDD.

Reply:
That’s certainly possible, it’ll just be the case of adding a conditional of “else, $device = windows 11 one, and if that exists then disable/enable”. Only thing I’d need is the driver’s friendly name (I’m not a Windows 11 user in anyway so either you’d need to paste that in or I can spin up a VM whenever I’m free to grab it)